NDTensors/src/array/permutedims.jl

```diff
@@ -1,6 +1,6 @@
 # NDTensors.permutedims
 function permutedims(::Type{<:Array}, M, perm)
-  return @strided Base.permutedims(M, perm)
+  return @strided A = Base.permutedims(M, perm)
```

Comment:
Let's leave a comment here to say we do this to evaluate the permutation avoid returning a `StridedView`.

Also I think this would be clearer for when we look back at the code:
```julia
  @strided Mperm = Base.permutedims(M, perm)
  return Mperm
```

Reply:
I updated the code and added a comment. Thanks!